### PR TITLE
Openfga dependency update

### LINKED ISSUE #13667: fix(deps): update github.com/openfga/api/proto digest to f5a955a #13649
fix(deps): update github.com/openfga/api/proto digest to f5a955a #13649

### PR DESCRIPTION
**Pull request by hamistao**
Closes #13667 by updating openfga to version 1.5.6.

### CODE REVIEW

File: lxd/db/openfga/openfga.go [hunk 1]
```diff
@@ -356,16 +359,30 @@ func (o *openfgaStore) ReadStartingWithUser(ctx context.Context, store string, f
...
-	// Validate the entity type.
 	entityType := entity.Type(filter.ObjectType)
 	err := entityType.Validate()
 	if err != nil {
-		return nil, fmt.Errorf("ReadStartingWithUser: Invalid entity type %q: %w", entityType, err)
+		return nil, fmt.Errorf("ReadUsersetTuples: Invalid object filter %q: %w", entityType, err)
+	}
+
+	// Check for type-bound public access exception.
+	if entityType == entity.TypeServer && filter.Relation == "can_view" {
```
Comment (tomponline) [inline lxd/db/openfga/openfga.go]: Do we have constants for "can_view" we can reference so if they ever change we can see where they are used?

**Review by tomponline — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -20,7 +20,7 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/gorilla/mux v1.8.1
 	github.com/gorilla/securecookie v1.1.2
-	github.com/gorilla/websocket v1.5.1
+	github.com/gorilla/websocket v1.5.3
```
Comment (tomponline) [inline go.mod:23]: Please can you limit the go mod update to just the ones relevant for this PR. Ta

Comment (tomponline) [reply go.mod:23]: We dont want to bump this one just yet.

File: lxd/db/openfga/openfga.go [hunk 3]
```diff
@@ -366,7 +366,7 @@ func (o *openfgaStore) ReadStartingWithUser(ctx context.Context, store string, f
 	}
 
 	// Check for type-bound public access exception.
-	if entityType == entity.TypeServer && filter.Relation == "can_view" {
+	if entityType == entity.TypeServer && filter.Relation == string(auth.EntitlementCanView) {
```
[the remark this answers is not in the document]
Comment (hamistao) [reply lxd/db/openfga/openfga.go:372]: @markylaing did :)

File: go.mod (hunk 2 above)
Comment (hamistao) [reply go.mod]: @tomponline Done!

**Review by tomponline — APPROVED** [review]

**Pull request merged**